projects/sandbox/export/export.py

```diff
@@ -103,22 +105,24 @@ def export(
     # of the network architecture, including preprocessor
     logging.info("Initializing model architecture")
     nn = architecture(num_ifos)
-    preprocessor = Preprocessor(num_ifos, sample_rate, fduration=fduration)
-    nn = torch.nn.Sequential(preprocessor, nn)
     logging.info(f"Initialize:\n{nn}")
 
-    # load in a set of trained weights and use the
-    # preprocessor's kernel_length parameter to infer
-    # the expected input dimension along the time axis
+    # load in a set of trained weights
     logging.info(f"Loading parameters from {weights}")
     state_dict = torch.load(weights, map_location="cpu")
     nn.load_state_dict(state_dict)
-    kernel_length = preprocessor.whitener.kernel_length.item()
-    logging.info(
-        f"Model will be exported with input kernel length of {kernel_length}s"
-    )
     nn.eval()
 
+    background_length = kernel_length - (fduration + 1)
```

Comment:
Going to suggest that we parameterize by 
`background_length` (amount of data used for calculating PSD), `inference_window_length` (amount of whitened data fed to the ResNet model) and `fduration`. 
So that `kernel_length = background_length + inference_window_length + fduration`.